scripts/load-docker-compose.sh

```diff
@@ -82,7 +24,21 @@ else
     sleep 10
 fi
 
+# Add path "/scripts/" in front of each non-option argument.
+for opt do
```

Comment:
This is obviously ok, but my preference is for explicitness.

Reply:
I wanted to avoid having two _identical_ command line parsing loops.  It's either that I add `/scripts/` in front of the pathnames in the wrapper script, or that I add them later in the actual loading script (`load-in-docker.sh`), in the `while` loop that parses the options.